metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -372,7 +368,9 @@ def _get_bigquery_log_entries(
             end_time=end_time,
         )
 
-        assert self.config.log_page_size is not None
+        if self.config.log_page_size is None:
```

Comment:
This is unreachable code, since the log_page_size will never be `None`, since it has a default value of `1000`.